CHANGELOG.md

```diff
@@ -15,6 +16,24 @@ The format is based on [Common Changelog](https://common-changelog.org/), and th
 [1.1.0]: https://github.com/sablier-labs/sdk/releases/tag/v1.1.0
 [1.0.0]: https://github.com/sablier-labs/sdk/releases/tag/v1.0.0
 
+## [2.0.0] - 2025-10-22
+
+#### Changed
+
+- `sablier.F00` now points to `sablier.evm.F00` for backward compatibility.
+  - `Sablier.Address` → `Sablier.EVM.Address`
+  - `Sablier.Chain` → `Sablier.EVM.Chain`
+  - `Sablier.Contract` → `Sablier.EVM.Contract`
+  - `Sablier.Release` → `Sablier.EVM.Release`
+  - `Sablier.Deployment` → `Sablier.EVM.Deployment`
+  - `Sablier.Protocol` → `Sablier.EVM.Protocol`
+  - `Sablier.Version` → `Sablier.EVM.Version`
```

Comment:
This is not a change as technically nothing changed for end users.

@gavriliumircea WDYT about deleting `CHANGELOG.md` and writing this only before releasing the npm package?

I don't find it worth it to maintain these `CHANGELOG.md` files between so many PR reviews.